apps/src/code-studio/components/GridEditor.jsx

```diff
@@ -191,6 +191,21 @@ export default class GridEditor extends React.Component {
     });
   };
 
+  /**
+   * Compute the number of unique map configurations, which is the product of the number
+   * of options for each Cell.
+   */
+  computeNumMaps = grid => {
+    let numMaps = 1;
+    grid.forEach(row => {
+      row.forEach(cell => {
+        let numPossibilitiesForCell = cell.getPossibleGridAssets().length;
```

Comment:
where is this defined?

Reply:
https://github.com/code-dot-org/maze/blob/master/src/cell.js#L105